homeworks/ekaterina.sharova_rogerosha/PokemonProject/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Masters Academy</title>
+        <link
+            href="./style.css"
+            rel="stylesheet"
+            type="text/css">
+    </head>
+    <body>
+        <h1>About me</h1>
+        <p>
+            Hi! My name is Katya and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like
+            HTML, CSS, JavaScript, and Git version control system.
+        </p>
+        <p>
+            This page was developed during the course <a href="https://www.mastersacademy.education" class="colortext"> 'Frontend for beginners' from Masters Academy in 2023</a>.
```

Comment:
Hey, @rogerosha, please find a decent name for the link class name, e.g. `link`)))